pkg/operator/staticresourcecontroller/static_resource_controller.go

```diff
@@ -58,9 +58,19 @@ type StaticResourceController struct {
 
 	eventRecorder events.Recorder
 
-	factory          *factory.Factory
-	restMapper       meta.RESTMapper
-	categoryExpander restmapper.CategoryExpander
+	factory *factory.Factory
+
+	conditionalResource *conditionalResource
```

Comment:
this should be a slice

Reply:
The [fields](https://github.com/openshift/library-go/pull/1095/files#diff-b91f253f2414046d3d35cc689c04410dfc00c7b8130a953ac59926118f9b8423R71) that this structure is holding is a slice. Do we still need make this slice?

Reply:
based on your current types, `conditionalResources []conditionalResource`.  I should be able to call `WithConditionalResources` multiple times.

Reply:
Ok. I can make that change.